docs/conf.py

```diff
@@ -28,10 +28,25 @@
 # it
 
 # Copy the docs from Cliquet inside these ones.
-destination = os.path.join(__HERE__, 'api', 'cliquet')
+# XXX how do we copy on 2.x ?
+destination = os.path.join(__HERE__, 'api', '1.x', 'cliquet')
```

Comment:
I was thinking that we could have the API implemented in Kinto package documented in the package documentation and only that. If we want to compare different version of APIs, we can do it by comparing documentation packages version as well. (Note that ReadTheDocs keep publishing all the different version of the Kinto packages)

We can probably also list the different APIs in another documentation repository (which could import them from the Kinto packages)